src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -681,7 +681,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Dim parameter As IParameterSymbol = boundParameterEqualsValue.Parameter
             Dim value As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundParameterEqualsValue.Value))
             Dim kind As OperationKind = OperationKind.ParameterInitializerAtDeclaration
-            Dim isInvalid As Boolean = DirectCast(boundParameterEqualsValue.Value, IOperation).IsInvalid
+            Dim isInvalid As Boolean = value.Value.IsInvalid
```

Comment:
>value.Value.IsInvalid [](start = 39, length = 21)

What is the point in creating Lazy(Of IOperation) if we get its value two lines later?